swagger-spec/logs/definition.yaml

```diff
@@ -0,0 +1,68 @@
+# Log Model Definition
+type: object
+title: Log
+required:
+  - id
+properties:
+  id:
+    description: 'The ID of the log.'
+    readOnly: true
+    type: string
+  date:
+    description: 'The date and time at which the log was created, as an iso8601 formatted timestamp.'
+    format: date-time
+    readOnly: true
+    type: string
+  action:
+    description: 'The type of action performed on the OSF. See description for full list of possible actions.'
+    readOnly: true
+    type: string
+  params:
+    description: 'The type of action performed on the OSF. See description for full list of possible actions.'
+    readOnly: true
+    type: string
+  links:
+    format: url
+    description: 'Links related to this log'
+    readOnly: true
+    type: string
+  node:
+    description: 'A relationship to the node associated with this log.'
+    readOnly: true
+    type: string
+  original_node:
+    description: 'A relationship to the original node that was associated with this log, in case this log was a remnant of when the original node was forked or registered.'
+    readOnly: true
+    type: string
+  user:
+    description: 'The user associated with this log'
+    readOnly: true
+    type: string
+  linked_node:
+    description: 'A node linked to this log'
+    readOnly: true
+    type: string
+  template_node:
+    description: 'Node used as a template for this log'
+    readOnly: true
+    type: string
+example:
```

Comment:
I think this might be a more complete example response (stolen from RPP's first page of logs).

```
data:
  relationships:
    node:
      links:
        related:
          href: https://api.osf.io/v2/nodes/ezcuj/
          meta: {}
    original_node:
      links:
        related:
          href: https://api.osf.io/v2/nodes/ezcuj/
          meta: {}
    user:
      links:
        related:
          href: https://api.osf.io/v2/users/jk5cv/
          meta: {}
  links:
    self: https://api.osf.io/v2/logs/4fc706a80b6e9118ef000122/
  attributes:
    date: '2012-05-31T05:50:32.083000'
    action: contributor_added
    params:
      preprint_provider: 
      params_node:
        id: ezcuj
        title: 'Reproducibility Project: Psychology'
      params_project: 
      contributors:
      - family_name: Nosek
        unregistered_name: 
        middle_names: A.
        given_name: Brian
        full_name: Brian A. Nosek
        active: true
        id: cdi38
  type: logs
  id: 4fc706a80b6e9118ef000122
```